src/main/java/chess/GetPossibleMovePiece.java

```diff
@@ -27,6 +29,31 @@ void possibleMovement(final ArrayList<Position> arrayPositions, final int column
         }
     }
 
+    /**
+     * Add all movements of the Horse
+     *
+     * @param arrayPositions
+     */
+    public void possibleMovementHorse(final ArrayList<Position> arrayPositions, final int column, final int row) {
```

Comment:
I think, you don't need composition between GetPossibleMovePiece adn Horse, only you could Implement this method into Horse class.

Reply:
For moviments of the horse is necessary "isPossibleToContinue", but this method is present in **GetPossibleMovePiece**.

I think if I implement in the horse class this is redundant.